pkgs/servers/mautrix-telegram/default.nix

```diff
@@ -1,14 +1,16 @@
-{ lib, python3, mautrix-telegram }:
+{ lib, python3, fetchFromGitHub, mautrix-telegram }:
 
 with python3.pkgs;
 
 buildPythonPackage rec {
   pname = "mautrix-telegram";
-  version = "0.5.2";
+  version = "0.6.0-rc1";
```

Comment:
I'm not sure whether we want -rc releases in nixpkgs, do we?

Reply:
It's only a draft PR to test the new version, I will update it when stable is out